mogwai/src/view/builder.rs

```diff
@@ -75,7 +75,7 @@ impl<T: IsDomNode> Default for ViewBuilder<T> {
 }
 
 impl<T: IsDomNode + AsRef<Node>> ViewBuilder<T> {
-    fn to_node(self) -> ViewBuilder<Node> {
+    pub fn to_node(self) -> ViewBuilder<Node> {
```

Comment:
I’m curious about the driver for this change. Only scanned through this but I didn’t see anything to necessitated it.

Reply:
It's a change for ergonomic/convenience. 

First, I'd like to be able to add a `Vec<ViewBuilder<Node>>`:

```rust
let children = todo!();
my_root.with(children);
```

But if any of those children are `ViewBuilder<Text>`, then it cannot be stored with the rest of the items in the vector. But alas, `Text` is `AsRef<Node>` and is also `JsCast` so we shoud be able to cast it. Maybe instead I should make the casting functions public, or write a trait for them.

What do you think?

Reply:
It has been a bit since I have been digging around but I remember thinking that I probably didn’t want to need access to the `Node` as a library consumer. Looking again this change does not do what I thought and is still giving a `ViewBuilder`, I was reacting to the function name without looking at the full signature (that’s what I get for looking quickly on mobile!). I definitely like the idea of being able to use a `Vec` of `ViewBuilder` to construct DOM trees but I don’t think I’m in it enough to have an opinion on the best way to achieve that though.